Sources/Core/Combine/Client+Requests.swift

```diff
@@ -0,0 +1,89 @@
+//
+// AppwiseCore
+// Copyright © 2022 Appwise
+//
+
+import Alamofire
+import Combine
+
+@available(iOS 13.0, *)
+public enum Return {
```

Comment:
Is this a normal name for this? Do other libraries use `Return` as a scope like this?